<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Style (Must restart)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="styleCombo"/>
       </item>
       <item>
        <widget class="QCheckBox" name="chordPreviewCheck">
         <property name="text">
          <string>Preview with all units (Ctrl+P)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pauseReseekCheck">
         <property name="text">
          <string>Spacebar stops instead of pauses</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="swapScrollOrientationCheck">
         <property name="text">
          <string>Swap vertical / horizontal scrolling</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="swapZoomOrientationCheck">
         <property name="text">
          <string>Swap vertical / horizontal zoom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="velocityDragCheck">
         <property name="text">
          <string>Adjust velocity with vertical drag</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoAddUnitCheck">
         <property name="text">
          <string>Auto-add unit with each new voice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unitPreviewClickCheck">
         <property name="text">
          <string>Click on unit to preview voice</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showWelcomeDialogCheck">
         <property name="text">
          <string>Show welcome dialog on startup</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>MIDI</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="midiInputPortCombo"/>
       </item>
       <item>
        <widget class="QCheckBox" name="autoAdvanceCheck">
         <property name="text">
          <string>Auto-step on note input (Ctrl+L)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="polyphonicMidiNotePreviewCheck">
         <property name="text">
          <string>Polyphonic note preview</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
